src/containers/Board/Board.js

```diff
@@ -41,74 +41,17 @@ class Board extends Component{
 
     moveCount = (board) => {
 
-        let moveCount = 0
-        for (let i = 0; i<board.length; i++){
-          for (let j = 0 ; j<board[i].length ; j++){
-            if (board[i][j]!==""){
-              moveCount++
-            }
-          }
-        }
-        this.setState({moves: moveCount})
-        return moveCount
-      };
-
-     getResult = (board,symbol) => {
-
-        let result = this.state.result.incomplete
-        if (this.moveCount(board)<5){
-           return {result}
-        }
-  
-        const succession =  (line) => {
-          return (line === symbol.repeat(3))
-        }
-  
-        let line
-        let winningLine=[]
-  
-        for (let i = 0 ; i<3 ; i++){
-          line = board[i].join('')
-          if(succession(line)){
-            result = symbol;
-            winningLine = [[i,0], [i,1], [i,2]]
-            return {result, winningLine};
-          }
-        }
-  
-        for (let j=0 ; j<3; j++){
-          let column = [board[0][j],board[1][j],board[2][j]]
-          line = column.join('')
-          if(succession(line)){
-            result = symbol
-            winningLine = [[0,j], [1,j], [2,j]]
-            return {result, winningLine};
+      let moveCount = 0
+      for (let i = 0; i<board.length; i++){
+        for (let j = 0 ; j<board[i].length ; j++){
+          if (board[i][j]!==""){
+            moveCount++
```

Comment:
Good job @hakoemmy 
However, `i think you could have made it better by impllmmmmm..... in this way

```js
const mute = 25
```